nexus/src/authz/omicron.polar

```diff
@@ -137,23 +137,42 @@ resource Silo {
 	];
 	roles = [ "admin", "collaborator", "viewer" ];
 
+	# permissions granted by this resource's roles
 	"list_children" if "viewer";
 	"read" if "viewer";
+	"create_child" if "collaborator";
```

Comment:
L143-144 just got moved (from L144, L146) -- they didn't really change.  I found it much clearer to reorder the rules in this block.  (There are also real changes here that I'll comment on separately.)